docs/modules/pipelines/pages/job-management.adoc

```diff
@@ -180,6 +182,13 @@ OK
 ALTER JOB hello-world RESUME;
 ----
 
+You can also update the configuration of a suspended job and resume it.
```

Comment:
Here, you can explain why it is necessary to increase the accumulation limit and furthermore how we come to this conclusion, so that it can become a "troubleshooting" section.

Reply:
I have not seen any support tickets raised for this type of issue, so you will need to help. I have added what I think might be appropriate but it is not based on any actual troubleshooting.

Reply:
Updated as requested.